htdocs/core/ajax/onlineSign.php

```diff
@@ -200,6 +200,16 @@
 
 				dol_syslog(__METHOD__, LOG_DEBUG);
 				$resql = $db->query($sql);
+				//Trigger propal signe en ligne
+				if (method_exists($object, 'call_trigger')) {
+					$user = new User($db);
+					$user->fetch($object->user_valid_id);
+					$result = $object->call_trigger('PROPAL_CLOSE_SIGNED_ONLINE', $user);
```

Comment:
There is no need to duplicate the trigger PROPAL_CLOSE_SIGNED into a variant PROPAL_CLOSE_SIGNED_ONLINE.
What you can do is just add the line
`$object->context = array('closedfromonlinesignature' => 'closedfromonlinesignature')`
before the 
$result = $object->call_trigger('PROPAL_CLOSE_SIGNED', $user); line 227